core/src/main/java/com/twosigma/beaker/core/rest/PluginServiceLocatorRest.java

```diff
@@ -727,6 +737,12 @@ else if (nginxRule.equals("ipython2"))
     }
     nginxConfig = nginxConfig.replace("%(plugin_section)s", pluginSection.toString());
     nginxConfig = nginxConfig.replace("%(extra_rules)s", this.nginxExtraRules);
+    if (this.hideZombieLogging) {
+      nginxConfig = nginxConfig.replace("%(catch_outdated_requests_rule)s",
+          this.CATCH_OUTDATED_REQUESTS_RULE);
+    } else {
+      nginxConfig = nginxConfig.replace("%(catch_outdated_requests_rule)s", "");
+    }
```

Comment:
better style:

nginxConfig = nginxConfig.replace("%(catch_outdated_requests_rule)s",
          this.hideZombieLogging ? this.CATCH_OUTDATED_REQUESTS_RULE : "");